pages/docs/components/form/button.mdx

```diff
@@ -78,6 +78,85 @@ value to `solid`, `ghost`, `outline`, or `link`.
 </Stack>
 ```
 
+### Button Colors
+
+Use the `colorScheme` prop to change the color scheme of the Button. You can set
+the value to `whiteAlpha`, `blackAlpha`, `gray`, `red`, `orange`, `yellow`,
+`green`, `teal`, `blue`, `cyan`, `purple`, `pink`, `linkedin`, `facebook`,
+`messenger`, `whatsapp`, `twitter` or `telegram`.
```

Comment:
Thank you for opening this PR, @rezamajidi! I'd just make this paragraph simpler by changing it to:

> Use the `colorScheme` prop to change the color scheme of the Button. You can set the value to any of the > color scales from your design system, like `whiteAlpha`, `blackAlpha`, `gray`, `red`, `blue`, or your custom 
> color scale.